canonical/canonical.com
# feat: Build /what-is-postgresql page

## Done

- Build the 'what-is-postgresql' page based on the [copydoc](https://docs.google.com/document/d/1GPCEnZgCfYvvPXn57iKL673PfNDkXkpY9fcGTxhNefg/edit#heading=h.5g6b1x38s47a) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=909-10435&node-type=FRAME&t=W6zUuA3DegagwprK-0)

### Flyby

- Remove unused comment from styles.scss

## QA

- Check out [the demo](https://canonical-com-1368.demos.haus/data/postgresql/what-is-postgresql)
- Check that the content matches that in the [copydoc](https://docs.google.com/document/d/1GPCEnZgCfYvvPXn57iKL673PfNDkXkpY9fcGTxhNefg/edit#heading=h.5g6b1x38s47a)
- Check the content is visible and logically laid out on all screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-13768
